app/src/main/java/com/github/polypoly/app/base/game/rules_and_lobby/kotlin/GameLobby.kt

```diff
@@ -1,7 +1,8 @@
-package com.github.polypoly.app.base.game.rules_and_lobby
+package com.github.polypoly.app.base.game.rules_and_lobby.kotlin
```

Comment:
Why include `kotlin` in the package? Will there be another programming language in the codebase that won't use this?

Reply:
This isn't the cleanest way to do it but we create `kotlin` packages when we want to separate back/frontend

Reply:
you could call them backend and frontend then :-)